frontend/src/container/MetricsApplication/Tabs/Overview/TopOperationMetrics.tsx

```diff
@@ -87,34 +77,19 @@ function TopOperationMetrics(): JSX.Element {
 
 	const queryTableData = data?.payload.data.newResult.data.result || [];
 
-	const renderColumnCell = useMemo(
-		() =>
-			getTableColumnRenderer({
-				columnName: 'operation',
-				renderFunction: (record: RowData): ReactNode => (
-					<ColumnWithLink
-						servicename={servicename}
-						minTime={minTime}
-						maxTime={maxTime}
-						selectedTraceTags={selectedTraceTags}
-						record={record}
-					/>
-				),
-			}),
-		[servicename, minTime, maxTime, selectedTraceTags],
-	);
+	if (isLoading) {
+		return <Spinner size="large" tip="Loading..." height="40vh" />;
+	}
 
 	if (errorMessage) {
 		return <div>{errorMessage}</div>;
 	}
 
 	return (
 		<QueryTable
-			title={title}
 			query={updatedQuery}
 			queryTableData={queryTableData}
 			loading={isLoading}
```

Comment:
if query table takes loading do we still need Spinner ?

Reply:
No need removing it.